heir_py/mlir_emitter.py

```diff
@@ -125,6 +242,10 @@ def emit_assign(self, assign):
       case ir.Global():
         self.globals_map[assign.target.name] = assign.value.name
         return ""
+      case ir.Var():
+        # FIXME: keep track of loop var, and assign
```

Comment:
Convert FIXME to TODO

Reply:
Ah, that's handled now in the loop emission with self.forward_name. Old note.